records_mover/db/postgres/copy_options/date_input_style.py

```diff
@@ -217,6 +359,15 @@ def upgrade_date_order_style(style: DateOrderStyle, hint_name: str) -> None:
         #
         #  postgres=#
         upgrade_date_order_style('MDY', 'dateformat')
+    elif dateformat == "DD/MM/YY":
+        #  postgres=# select date '02/01/99';
+        #      date
+        #  ------------
+        #   1999-02-01
+        #  (1 row)
+        #
+        #  postgres=#
+        upgrade_date_order_style('DMY', 'dateformat')
     elif dateformat is None:
         # null implies that that date format is unknown, and that the
         # implementation SHOULD generate using their default value and
```

Comment:
In general you can see this file is very case-based.  I think it'd both be less exhausting as a piece of code and more helpful for folks if it were turned into a more dynamic function that didn't need to check every single supported case, but instead did something more dynamic ("is DD before MM in the string in front of me?).  That'd also be one less piece to improve before we can advertise general support for date/time format strings in our hints and loosen the Records Spec accordingly.